impl/src/main/java/org/jboss/weld/bootstrap/events/ProcessSimpleInjectionTarget.java

```diff
@@ -45,6 +45,7 @@ public ProcessSimpleInjectionTarget(BeanManagerImpl beanManager, AnnotatedType<X
     public void setInjectionTarget(InjectionTarget<X> injectionTarget) {
         checkWithinObserverNotification();
         this.injectionTarget = injectionTarget;
+        BootstrapLogger.LOG.setInjectionTargetCalled(getReceiver().getClass(), injectionTarget.getClass());
```

Comment:
I find it quite nice :-). One can see the class name and the hashcode of the instance. Don't forget it's only a debug message, it does not have to be nice.